argobots.spec

```diff
@@ -1,22 +1,33 @@
 Name: argobots
-Version: 1.0
-Release: 1%{?dist}
+
+%global major 1
+%global minor 1
+%global prerelease a1
+%global tag %{major}.%{minor}%{?prerelease}
+
+Version: %{major}.%{minor}%{?prerelease:~%{prerelease}}
+Release: 1.g7fd1987%{?dist}
 Summary: Lightweight, low-level threading and tasking framework
 Group: System Environment/Libraries
 License: UChicago Argonne, LLC -- Argobots License
 Url: http://www.argobots.org/
 #Source: https://api.github.com/repos/pmodels/$(NAME)/tarball/31703b1
-Source: https://github.com/pmodels/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.gz
+Source: https://github.com/pmodels/%{name}/releases/download/v%{tag}/%{name}-%{tag}.tar.gz
+# patch to gather all changes from v1.1a1 tag up to ULTs stack unwinding commit
+Patch0: v1.1a1..7fd1987.patch
 
 BuildRequires: pkgconfig
 
 # to be able to generate configure if not present
 BuildRequires: autoconf, automake, libtool
 
+# need libunwind if configuring symbolic ULTs stack dumps feature
+BuildRequires: libunwind-devel
+
 %ifarch x86_64
-%global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static --enable-psm --enable-psm2
+%global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static --enable-psm --enable-psm2 --enable-stack-unwind
 %else
-%global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static
+%global configopts --enable-sockets --enable-verbs --enable-usnic --disable-static --enable-stack-unwind
 %endif
```

Comment:
Get rid of this block:
```suggestion
```

and add the `--enable-stack-unwind` onto the end of line 74.  `%configopts` is not actually supported in this specfile and is a remanent of using the libfabric specfile as a template for this specfile and not completly removing the _unneded-at-the-time_ `%configopts` support it had.

You will have to re-run daos-stack/daos#4590 to revalidate the re-build with the libunwind included against current master.  I really don't expect it to fail, but we should do the due diligence and make sure we don't disrupt the rest of the team.